wiki/PlayersManual.md

```diff
@@ -376,6 +376,7 @@ Search (for planet, system, ship, or outfit) | F
 Rotate jump destination | J or Tab
 Add jump destination | Shift + J or Tab
 Compare (two ships or outfits) | Shift + Click
+Toggle starry map mode | x
```

Comment:
x? Not anymore.
If we're adding this, we should probably have the keys for switching to the shipyard, outfitter, port, and mission map panels.